DocumentedExamples/Barotropic_Streamfunction_model_agnostic.ipynb

```diff
@@ -0,0 +1,1660 @@
+{
```

Comment:
<p>this figure looks weird... </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>